Dice/Dice/Program.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Dice
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            uint numberOfDices = 2;
+            Player player = new Player("John", numberOfDices);
```

Comment:
The purpose of a constructor is fields initialization. Here you just create a new object. It isn't obvious that this piece of code will perform any action.